src/components/specific/files/files-table/file-tags-cell/FileTagsCell.vue

```diff
@@ -33,6 +33,7 @@
           ref="tagList"
           class="file-tags-cell-pills__over"
           width="150px"
+          style="top: -9px"
```

Comment:
Can we can use the `yOffset` parameter to solve this ?
I'm not sure to understand the problem, let me know if you want to discuss about it. :)